test/paper-dropdown-menu-light.html

```diff
@@ -40,7 +38,7 @@
   <test-fixture id="PreselectedDropdownMenu">
     <template>
       <paper-dropdown-menu-light no-animations>
-        <paper-listbox class="dropdown-content" selected="1">
+        <paper-listbox slot="dropdown-content" class="dropdown-content" selected="1">
```

Comment:
the 1.x variant will load `paper-menu-button v1.x` which will distribute via `<content>`, hence the class is needed for that